modules/provision/volume.go

```diff
@@ -49,7 +49,9 @@ func volumeProvision(ctx context.Context, reservation *Reservation) (interface{}
 		return path, nil
 	}
 
-	return storageClient.CreateFilesystem(reservation.ID, config.Size*gigabyte, modules.DeviceType(config.Type))
+	_, err = storageClient.CreateFilesystem(reservation.ID, config.Size*gigabyte, modules.DeviceType(config.Type))
+	// nothing to return to BCDB
```

Comment:
Why, shouldn't this return the ID of the volume ?

Reply:
I didn't send it cause the ID is actually the reservation ID. But this is an implementation detail and that could change in the future. so yep I need to send the volume ID back.